### DISCUSSION TITLE
Beep or Tune when Finished

### DISCUSSION THREAD

**Discussion by KevinWelay** [Jyers/Marlin#1078] [Ideas]
Hello Not sure if possible, I know it can beep any chance to add a finished sound or beeps so i can know when its done and have a few different ones so you know which machine it is 

thanks 
Great work

**Comment by adam3654** [comment]
See the [Marlin M300 reference](https://marlinfw.org/docs/gcode/M300.html)

You can add the beeps to your end g code in your slicer. Not everyone wants beeps.
